packages/core/integration-tests/test/css.js

```diff
@@ -77,19 +77,12 @@ describe('css', () => {
       {
         name: 'entry.js',
         type: 'js',
-        assets: [
-          'bundle-url.js',
-          'cacheLoader.js',
-          'css-loader.js',
-          'entry.js',
-          'js-loader.js',
-        ],
+        assets: ['bundle-url.js', 'cacheLoader.js', 'entry.js', 'js-loader.js'],
       },
       {
         type: 'js',
         assets: ['esmodule-helpers.js', 'index.js'],
       },
-      {name: 'Foo.css', type: 'css', assets: ['foo.css']},
       {name: 'entry.css', type: 'css', assets: ['foo.css', 'main.css']},
```

Comment:
I think this was actually a similar case to the one I originally described in the description. foo.css doesn't need to be duplicated because it's already loaded in a parallel bundle